### PR TITLE
[FOGL-2870] Interface to install available filter plugins

### PR DESCRIPTION
**Pull request by monika-Dianomic**

### CODE REVIEW

File: src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts [hunk 1]
```diff
@@ -21,12 +21,27 @@ export class AddFilterWizardComponent implements OnInit {
   public isValidName = true;
   public payload: any;
   public selectedPluginDescription = '';
-
+  public pluginData = [];
   public filesToUpload = [];
+  public installNewPlugin = false;
+  private REQUEST_TIMEOUT_INTERVAL = 2000;
```
Comment (praveen-garg) [inline src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts]: for install plugin, please make it 5sec. and  test with expression filter (on RPi) and GUI on another machine

File: src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts [hunk 2]
```diff
@@ -166,6 +249,44 @@ export class AddFilterWizardComponent implements OnInit {
...
+    }
+    const pluginData = {
+      format: 'repository',
+      name: `foglamp-filter-` + pluginName,
+      version: ''
+    };
+
+    /** request started */
+    this.ngProgress.start();
+    this.alertService.activityMessage('installing...', true);
```
Comment (praveen-garg) [inline src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts]: `installing ...` 

space before `...`

File: src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts [hunk 3]
```diff
@@ -297,6 +417,11 @@ export class AddFilterWizardComponent implements OnInit {
           this.alertService.error(error.statusText);
         }
       });
+    if (this.installNewPlugin) {
```
Comment (praveen-garg) [inline src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts]: can you please add a comment in code, why there is timeout and moveNext on what conditions?

File: src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts (hunk 1 above)
Comment (monika-Dianomic) [reply src/app/components/core/filter/add-filter-wizard/add-filter-wizard.component.ts]: I have already tested it on RPi and GUI on another machine.

**Pull request merged**